### PR TITLE
Added 4 examples in TypeScript (see below)

### PR DESCRIPTION
**Pull request by Ray6464**
Added the following 4 example scripts to TypeScript
  - Reversing a String
  - Capitalize

Sorry, didn't saw the requirements below until I already added these scripts to my fork. Will look out for them next time.

### Code Snippets

- [x] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  
### Documentation

- [x] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [x] I followed the applicable article template
  - [x] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [ x I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [x] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [x] I names the pull request using `Added/Updated <Language>/<Project> Testing` format
- [ ] I followed the testinfo template, if applicable??

## Notes

Feel free to describe what you added or updated.

### CODE REVIEW

File: archive/j/javascript/exportExample.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+//exporting to importExample.ts
```
Comment (jrg94) [inline archive/j/javascript/exportExample.ts]: We currently don't support the export and import examples. Any chance you'd be willing to put together some documentation together for these solutions? You can see some examples here: https://sample-programs.therenegadecoder.com/projects/

File: archive/t/typescript/capitalize.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const myStr: string = "hello world";
+
+const capitalize = (str: string = "no input was provided") => str[0].toUpperCase() + str.slice(1);
+
+console.log(capitalize(myStr));
```
Comment (jrg94) [inline archive/t/typescript/capitalize.ts]: Looks good!

File: archive/j/javascript/exportExample.ts (hunk 1 above)
Comment (Ray6464) [reply archive/j/javascript/exportExample.ts]: Sure, I'll work on a brief documentation for import and export for JavaScript. Meanwhile I'll remove them from the PR.

**Comment by Ray6464** [comment]
Making a new Pull request

**Comment by Ray6464** [comment]
Making a new Pull request

**Pull request closed without merging**